app/views/messages/index.html.erb

```diff
@@ -1,19 +1,23 @@
-<div id="allmess">
-    <div class="container">
-        <% if @messages %>
-        <p>All Messages</p>
-        <% @messages.each do |mess| %>
-        <div id="mess">
-            <div class="white"><%= mess.emailadd %></h3>
-        </div>
-        <br>
-        <div><%= mess.content %></div>
-    </div>
-
-    <hr>
+<div id="allmess" style="overflow: scroll;">
+  <div class="container">
+    <% if @messages.present? %>
+      <p>All Messages</p>
+        <% @messages.each do |m| %>
```

Comment:
`m`. M & M? Do the right thing. Don't do one letter variables.

This is the `m` variable right now 👇 
![](http://i.imgur.com/EXqCn3y.gif)